brave/brave-core
# Add Neon EVM to preloaded chains + remove Fantom

This PR does three things:
1. Adds Neon EVM Mainnet to preloaded networks.
2. Removes Fantom Opera from preloaded networks (see internal [Slack discussion](https://bravesoftware.slack.com/archives/C023VS4HJ6Q/p1690554894315509)).
3. Changed the order of preloaded networks based on TVL from https://defillama.com/chains/EVM (except Aurora).


Resolves https://github.com/brave/brave-browser/issues/31760
Resolves https://github.com/brave/brave-browser/issues/32085

Token list update adding Neon logo: https://github.com/brave/token-lists/pull/75



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

### Testing addition of Neon EVM

1. Neon EVM shows up on Portfolio.
2. Neon logo is visible
3. The price of `NEON` is accurate

### Testing removal of Fantom Opera

(steps)

1. Open a Brave Browser build that does not contain changes in this PR.
2. Visit matcha.xyz and connect Brave Wallet.
3. Switch to Fantom
4. Open a Brave Browser build that contains this PR. Make sure the two browsers share the same user prefs.
5. Visit `brave://settings/wallet/networks`
6. Fantom Opera should be added as a custom chain
7. Visit matcha.xyz again and connect Brave Wallet.
8. The connection should default to Fantom Opera network.
